docs/source/tutorials/Index.md

```diff
@@ -1,23 +1,23 @@
 Index.md
 
-All MAAP data tutorials can be found in this folder. 
+All MAAP data tutorials can be found in this folder.
 
 Contents:
 1. GEDI
 
 	a. L1A
-	
+
 	b. L1B
-	
-	c. L2A 
-	
+
+	c. L2A
+
 	d. L2B
-		In this tutorial we use the integrated Earthdata serch function in MAAP to search for some GEDI L2B data for an area of interest. 
-		The data is then downloaded and prepared for analysis. We perform a spatial subset on the downloaded data to reduce data volumes and create 
+		In this tutorial we use the integrated Earthdata serch function in MAAP to search for some GEDI L2B data for an area of interest.
+		The data is then downloaded and prepared for analysis. We perform a spatial subset on the downloaded data to reduce data volumes and create
 		some basic plots.
-		
+
 	e. L3
-	
+
 	f. L4A
-	
+
```

Comment:
Remove this file completely

Reply:
It's gone

Reply:
Also removed other files that you mentioned. Thank you for catching that.